packages/data/tests/unit/adapters/bard-facts-test.js

```diff
@@ -153,6 +162,21 @@ module('Unit | Bard facts Adapter', function(hooks) {
       'm1,m2',
       '_buildMetricsParam built the correct string for many metrics'
     );
+
+    let duplicateMetrics = {
+      metrics: [
+        { metric: 'm1' },
+        { metric: 'm2' },
+        { metric: 'm1' },
+        { metric: 'r', parameters: { p: '123', as: 'm1' } },
+        { metric: 'r', parameters: { p: '123', as: 'm2' } }
```

Comment:
Do we need to add or already have a test that has the same metric with different params?

Reply:
will add